settings.yaml

```diff
@@ -84,6 +84,12 @@ vectorstore:
 nodestore:
   database: simple
 
+milvus:
+  uri: "./milvus_llamaindex.db"
+  token: ""
+  collection_name: "milvus_db"
+  overwrite: True
```

Comment:
Can you remove `"` when it is not required? Also change `True` by `true` to be aligned with rest of settings

Reply:
No problem, thanks for the comments.